modules/apps/calendar/calendar-web-test/src/testFunctional/tests/AddCalendarAccessibility.testcase

```diff
@@ -0,0 +1,74 @@
+@component-name = "portal-calendar"
+definition {
+
+	property testray.main.component.name = "Calendar";
+
+	var pageName = "Calendar Page";
+	var portletName = "Calendar";
+
+	setUp {
+		var portalURL = PropsUtil.get("portal.url");
+
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+
+		task ("Given calendar portlet") {
+			JSONLayout.addPublicLayout(
+				groupName = "Guest",
+				layoutName = "Calendar Page");
+
+			JSONLayout.updateLayoutTemplateOfPublicLayout(
+				groupName = "Guest",
+				layoutName = "Calendar Page",
+				layoutTemplate = "1 Column");
+
+			JSONLayout.addWidgetToPublicLayout(
+				groupName = "Guest",
+				layoutName = "Calendar Page",
+				widgetName = "Calendar");
+
+			Navigator.openSpecificURL(url = "${portalURL}/web/guest/calendar-page");
+		}
+	}
+
+	tearDown {
+		var testLiferayVirtualInstance = PropsUtil.get("test.liferay.virtual.instance");
+
+		if (${testLiferayVirtualInstance} == "true") {
+			PortalInstances.tearDownCP();
+		}
+		else {
+			Navigator.gotoPage(pageName = "Calendar Page");
+
+			JSONCalendar.tearDown();
+
+			CalendarConfiguration.tearDown();
+
+			PagesAdmin.tearDownCP();
+		}
+	}
+
+	@description = "LPS-185844 Verifies that 'More Options' button is available after applying 200% zoom in to the page."
```

Comment:
"LPS-185844 Verifies that the last scrollable element is not hidden by modal footer at 200% page zoom."